dev/com.ibm.ws.security.javaeesec.internal/original.bnd

```diff
@@ -0,0 +1,13 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+Bundle-Name: WAS Java EE Security Internal 1.0
+Bundle-SymbolicName: com.ibm.ws.security.javaeesec.internal.1.0
```

Comment:
Since this is a new bundle and project, it should be called io.openliberty..... to follow the naming guidelines.  Personally I would move it to `io.openliberty.security.javaeesec.internal` and have that be the bundle name for original.bnd and hav eit be `io.openliberty.security.javaeesec.internal.jakarta` for the transformed bundle.